CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```

Comment:
Start with adding first description of what you have done in this pull request instead of a placeholder text.